lib/rubocop/cop/rspec/cop.rb

```diff
@@ -5,11 +5,9 @@ module Cop # rubocop:disable Style/Documentation
     # Clone of the the normal RuboCop::Cop::Cop class so we can rewrite
     # the inherited method without breaking functionality
     class WorkaroundCop
-      # Remove the cop inherited method to be a noop. Our RSpec::Cop
+      # Remove the Cop.inherited method to be a noop. Our RSpec::Cop
       # class will invoke the inherited hook instead
-      class << self
-        remove_method :inherited
-      end
+      def self.inherited(*) end
```

Comment:
`WorkaroundCop` uses `remove_method` to avoid MRI warning.
See https://github.com/backus/rubocop-rspec/pull/436

Could you use `undef`? Like:

```ruby
class WorkaroundCop
  # Remove the Cop.inherited method to be a noop. Our RSpec::Cop
  # class will invoke the inherited hook instead
  class << self
    undef inherited
    def inherited(*) end
  end
```

It works well on JRuby, and MRI does not warn about it.